### PR TITLE
Support for 2019 CU4 in build reference

### PR DESCRIPTION
**Pull request by alevyinroc**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [X] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
Update build reference for latest SQL Server 2019 CU

### Approach

Updates internal buildref

### CODE REVIEW

**Comment by wsmelton** [comment]
You need to update the date property at the top of the build file.

**Comment by alevyinroc** [comment]
I did that in #6436 but maybe that didn't get merged in yet? Fixed now.

**Comment by wsmelton** [comment]
I'm not referring to the date of SQL Server. I reverted the change you made because we do not track the extended support date, **only** the mainstream support.

The json file itself on line 2 or 3 has a date the file was last touched. That date has to be updated for it to get picked up properly by the commands. Otherwise it will think it is out dated.

File: bin/dbatools-buildref-index.json [hunk 1]
```diff
@@ -4181,7 +4181,7 @@
         {
             "Version": "15.0.2000",
             "SP": "RTM",
-            "SupportedUntil": "2025-01-07"
+            "SupportedUntil": "2030-01-08"
```
Comment (wsmelton) [inline bin/dbatools-buildref-index.json]: This date should be reverted. We do not track the extended support date, only the mainstream.

**Comment by wsmelton** [comment]
You can review #6356 for example of how this file is supposed to be udpated.

File: bin/dbatools-buildref-index.json [hunk 2]
```diff
@@ -4201,6 +4201,11 @@
             "CU": "CU3",
             "Version": "15.0.4023",
             "KBList": "4538853"
+        },
+        {
+            "CU": "CU4",
+            "Version": "15.0.4033",
+            "KBList": "KB4548597"
```
Comment (wsmelton) [inline bin/dbatools-buildref-index.json:4208]:
```suggestion
            "KBList": "4548597"

```

Comment (wsmelton) [reply bin/dbatools-buildref-index.json:4208]: This is why the test is failing.

**Pull request merged**